geotrek/common/parsers.py

```diff
@@ -13,7 +13,7 @@
 import xlrd
 import xml.etree.ElementTree as ET
 from functools import reduce
-from collections import Iterable
+from collections.abc import Iterable
```

Comment:
**Suggestion:** The `collections.abc` module is used for abstract base classes related to collections. If you are using `Iterable` just for type checking, consider using `typing.Iterable` instead. [best practice]
```suggestion
from typing import Iterable
```